src/main/java/core/basesyntax/Plane.java

```diff
@@ -7,5 +7,50 @@
  */
 
 public class Plane {
+    private String planeModel;
+    private String planeEngineType;
+    private int planeSize;
+    private int planeSpeed;
+    private PlaneBuilder planeBuilder;
```

Comment:
should Plane contain a field with PlaneBuilder

Reply:
done